<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogIdentify</class>
 <widget class="QWidget" name="DialogIdentify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>识别</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#centralWgt
{
	background-image: url(:/image/ui_background_on_iden.png);
}</string>
  </property>
  <widget class="QFrame" name="centralWgt">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">/*QWidget#wgtLeft
{    
	background-image: url(:/image/ui_iden_left_irisking.png);
}*/


QRadioButton:checked{
    color: blue;
    font: bold;
}

QRadioButton::indicator:unchecked {
	image: url(:/image/radiobuttonUnchecked_32pt.png);
 }

QRadioButton::indicator:checked {
	image: url(:/image/radiobuttonChecked_32pt.png);
 }

QWidget#wgtNetStatus
{
	background-image: url(:/image/ui_network_state.png);
}

QToolButton#toolBtnBell{
        border-top-left-radius: 3px;
        border-bottom-left-radius: 3px;  
	    background-image: url(:/image/ui_bell_normal.png);
    }
QToolButton#toolBtnBell:pressed{
	    background-image: url(:/image/ui_bell_pressed.png);
    }

QToolButton#toolBtnLogin{
        border-top-left-radius: 3px;  
        border-bottom-left-radius: 3px; 
	    background-image: url(:/image/ui_login_normal.png);
    }
QToolButton#toolBtnLogin:pressed{
	    background-image: url(:/image/ui_login_pressed.png);
    }

QLabel#labBorder
{
	background-image: url(:/image/ui_video_border.png);
}

QLabel#labLogo
{
	background-image: url(:/image/ui_iden_left_irisking.png);
}

QLabel#labStatu
{
    color: rgb(0, 0, 0);
    border-radius: 20px;
    padding: 6px;
    font: bold 26pt &quot;浪漫雅圆&quot;;
	border-image: url(:/image/ui_bar_wait.png);
}

QLabel#labDate
{
	border-image: url(:/image/ui_date_background.png);
    color: rgb(30, 30,30);
}

QLabel#labEye
{
     border-radius: 7px;
}

*{
   color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="labelSleep">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>138</y>
      <width>498</width>
      <height>373</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labEye">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>104</y>
      <width>664</width>
      <height>498</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(149, 201, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labDate">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>68</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garuda</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> 星期三   2014-03-12</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QWidget" name="wgtLeftWeatherInfo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>105</y>
      <width>180</width>
      <height>498</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="labWeatherIMG">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../ikembx00.qrc">:/image/ui_weather_1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labCityName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>北京</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labTemp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温度：N/A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labWeather">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>气象信息：N/A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labWinp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>风向：N/A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="wgtRight" native="true">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>1</y>
      <width>180</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QToolButton" name="toolBtnLogin">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>330</y>
       <width>165</width>
       <height>110</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolBtnBell">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>165</width>
       <height>110</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labCountDown">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>336</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
background-image: url();
border-radius: 6px; </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="wgtNetStatus" native="true">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>520</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labNetStateVal">
        <property name="minimumSize">
         <size>
          <width>63</width>
          <height>63</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>63</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labNetStatus">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>网络状态</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url();</string>
     </property>
     <property name="text">
      <string>帧率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="labIsLogin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>120</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url();</string>
     </property>
     <property name="text">
      <string>考勤识别</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveImage">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>470</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 127);
border-radius:8;</string>
     </property>
     <property name="text">
      <string>存图</string>
     </property>
    </widget>
    <zorder>toolBtnBell</zorder>
    <zorder>toolBtnLogin</zorder>
    <zorder>labCountDown</zorder>
    <zorder>wgtNetStatus</zorder>
    <zorder>labCount</zorder>
    <zorder>labIsLogin</zorder>
    <zorder>btnSaveImage</zorder>
   </widget>
   <widget class="QWidget" name="wgtTime" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>7</y>
      <width>380</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labHour">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labMinute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labSecond">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labBorder">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>104</y>
      <width>666</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labStatu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>510</y>
      <width>491</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>320</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>浪漫雅圆</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>状态</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="wgtLeftLogo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>105</y>
      <width>180</width>
      <height>498</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labLogo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="saveImageStatus">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>560</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>104</y>
      <width>664</width>
      <height>498</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>649</width>
      <height>488</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>666</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>labelSleep</zorder>
   <zorder>labEye</zorder>
   <zorder>labDate</zorder>
   <zorder>wgtLeftWeatherInfo</zorder>
   <zorder>wgtRight</zorder>
   <zorder>wgtTime</zorder>
   <zorder>wgtLeftLogo</zorder>
   <zorder>saveImageStatus</zorder>
   <zorder>label</zorder>
   <zorder>labBorder</zorder>
   <zorder>labStatu</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../ikembx00.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolBtnLogin</sender>
   <signal>clicked()</signal>
   <receiver>DialogIdentify</receiver>
   <slot>selectLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBtnBell</sender>
   <signal>clicked()</signal>
   <receiver>DialogIdentify</receiver>
   <slot>sendBell()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendBell()</slot>
  <slot>selectLogin()</slot>
 </slots>
</ui>
